retrofit-converters/logansquare/src/main/java/retrofit/LoganSquareConverterFactory.java

```diff
@@ -0,0 +1,21 @@
+package retrofit;
+
+import com.bluelinelabs.logansquare.JsonMapper;
+import com.bluelinelabs.logansquare.LoganSquare;
+
+import java.lang.reflect.Type;
+
+/**
+ * A {@linkplain Converter.Factory converter} which uses LoganSquare as its conversion backbone.
+ *
+ * @author Marcel Schnelle (aurae)
+ * @see <a>https://github.com/bluelinelabs/LoganSquare</a>
+ */
+public final class LoganSquareConverterFactory implements Converter.Factory {
+
+	@Override public Converter<?> get(Type type) {
+		// Obtain the JsonMapper for the type and serve a Converter
+		JsonMapper<?> mapper = LoganSquare.mapperFor((Class<?>) type);
```

Comment:
You absolutely can use it on Lists (and String-keyed Maps, for that matter). I'm not sure what I was thinking here. I will update the PR with List and Map support asap.